receiver/awscloudwatchreceiver/logs.go

```diff
@@ -120,31 +124,51 @@ func newLogsReceiver(cfg *Config, logger *zap.Logger, consumer consumer.Logs) *l
 		autodiscover = nil
 	}
 
+	startTime := time.Unix(0, 0)
+	if cfg.Logs.StartFrom != "" {
+		parsedTime, err := time.Parse(time.RFC3339, cfg.Logs.StartFrom)
+		if err != nil {
+			settings.Logger.Error("Unable to parse start time", zap.Error(err))
```

Comment:
I think we could also perform this validation at the config [Validate()](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/39007/files#diff-7f9a432576bab2809be1d64e26779165ea6b4f6bba2386935f2babcfac0687c1R68) function.